doc/03_reference/cs_registers.rst

```diff
@@ -555,10 +555,10 @@ Machine Architecture ID (marchid)
 
 CSR Address: ``0xF12``
 
-Reset Value: ``0x0000_0016``
+Reset Value: ``CSR_MARCHID_VALUE``
 
 Use the ``CSR_MARCHID_VALUE`` parameter in :file:`rtl/cve2_pkg.sv` to change the fixed value.
```

Comment:
yes but MARCHID_VALUE is equal to MARCHID, see https://github.com/davideschiavone/cve2/blob/fix12/rtl/cve2_pkg.sv#L564

Reply:
right, but would you think a user should change MARCHID or CSR_MARCHID_VALUE definition? To me it seems they should use MARCHID

Reply:
MARCHID and MARCHID_VALUE are the same value, and it's a read only register. If it is easier for the user to read "35" or MARCHID I can change it - let me know

Reply:
my point is, if an integrator wants a different value for MARCHID, this will be done like:

```
module cve2_top import cve2_pkg::*; #(
  parameter MARCHID = 32'd99,
  ...
)
```

You essentially added a parameter, which is good, but it was not reflected in the documentation. There shouldn't be a need to modify inside the pkg anymore

Reply:
Oh I see, I can move it to `localparam` , what do you think?

Reply:
Ok then as soon as you approve the PR I merge :) thanks